CMakeLists.txt

```diff
@@ -988,6 +988,7 @@ endif()
 # Enable components
 set(CPACK_DEB_COMPONENT_INSTALL ON)
 set(CPACK_RPM_COMPONENT_INSTALL ON)
+set(CPACK_productbuild_COMPONENT_INSTALL ON)
```

Comment:
Should this be capitalised?

Reply:
I think this should be corresponding to generator name.

On macOS, supported generators are:

```console
% cpack -G unknown
CPack Error: Could not create CPack generator: unknown

Generators
  7Z                           = 7-Zip file format
  Bundle                       = Mac OSX bundle
  DragNDrop                    = Mac OSX Drag And Drop
  External                     = CPack External packages
  IFW                          = Qt Installer Framework
  NSIS                         = Null Soft Installer
  NSIS64                       = Null Soft Installer (64-bit)
  NuGet                        = NuGet packages
  PackageMaker                 = Mac OSX Package Maker installer
  STGZ                         = Self extracting Tar GZip compression
  TBZ2                         = Tar BZip2 compression
  TGZ                          = Tar GZip compression
  TXZ                          = Tar XZ compression
  TZ                           = Tar Compress compression
  TZST                         = Tar Zstandard compression
  ZIP                          = ZIP file format
  productbuild                 = Mac OSX pkg
```